pages/linux/httpie.md

```diff
@@ -0,0 +1,31 @@
+# httpie
+
+> A human-friendly CLI HTTP tool.
+
+- GET method (default method with no request data):
+
+`http {{https://api.github.com/users}}`
+
+- POST method (default method with request data):
+
+`http {{example.org}} {{hello=World}}`
+
+- POST method with redirected input:
```

Comment:
Perhaps `Send a POST request with a body from stdin` would read better?

Reply:
I think `redirected input` is different from `stdin`

Reply:
Not that I'm aware of. `stdin` is the standard input, which is what you're redirecting when you use the `<` symbol.